packages/allure-mocha/test/fixtures/runner.js

```diff
@@ -0,0 +1,55 @@
+// cjs: const { Glob } = require("glob");
+// cjs: const path = require("path");
+// cjs: const Mocha = require("mocha");
+// cjs: const AllureReporter = require("allure-mocha");
+// esm: import { Glob } from "glob";
+// esm: import path from "path";
+// esm: import { fileURLToPath } from "url";
+// esm: import Mocha from "mocha";
+// esm: import AllureReporter from "allure-mocha";
+
+// cjs: const dirname = __dirname;
+// esm: const dirname = path.dirname(fileURLToPath(import.meta.url));
+
+let emitFiles = false;
+let parallel = false;
+const requires = [];
+
+const sepIndex = process.argv.indexOf("--");
+const args = sepIndex === -1 ? [] : process.argv.splice(sepIndex);
+for (let i = 0; i < args.length; i++) {
```

Comment:
Use `forEach` instead

Reply:
Can't use forEach: there is a top-level await in the ESM version. But I've changed the loop to `for (const arg of args) ...`.